lib/commands/project.ts

```diff
@@ -196,13 +203,13 @@ export class InitCommand extends InitProjectCommandBase {
 	public execute(args: string[]): IFuture<void> {
 		return (() => {
 			if(this.isProjectType("Apache Cordova").wait()) {
-				this.$logger.info("Attempting to initialize Apache Cordova project.");
+				this.$logger.info("Attempting to initialize Cordova project.");
 				this.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
 			} else if(this.isProjectType("NativeScript").wait()) {
-				this.$logger.info("Attempting to initialize  NativeScript project.");
+				this.$logger.info("Attempting to initialize NativeScript project.");
 				this.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.NativeScript).wait();
 			} else if(this.isProjectType("Mobile Website").wait()) {
-				this.$logger.info("Attempting to initialize Mobile Website project.");
+				this.$logger.info("Attempting to initialize MobileWebsite project.");
```

Comment:
Because new output is "Successfully initialized MobileWebsite project.". It looks strange in the following way:
Attempting to initialize Mobile Website project.
Successfully initialized MobileWebsite project.

As the code for successfull initialization is using TARGET_FRAMEWORK_IDENTIFIERS, which is string, I decided to change it here to make the output consistent